README.md

```diff
@@ -44,6 +44,13 @@ You'll get an error in the end, but an executable will nevertheless be created i
 zip -j runtime.zip bootstrap /path/to/lumo runtime.cljs
```

Comment:
Maybe we could split up the readme into a "using make" section and a "manual" section?